icu4c/source/i18n/unicode/gregocal.h

```diff
@@ -619,6 +619,15 @@ class U_I18N_API GregorianCalendar: public Calendar {
      */
     virtual void handleComputeFields(int32_t julianDay, UErrorCode &status) override;
 
+#ifndef U_HIDE_INTERNAL_API
+    /**
+     * The year in this calenar is counting from 1 backward if the era is 0.
```

Comment:
s/calenar/calendar. I see this in a couple other places in this PR too.

Reply:
sorry